packages/salesforcedx-vscode-lwc/snippets/lwc.json

```diff
@@ -0,0 +1,224 @@
+    "prefix": "lwc-nav-object-page",
+    "description": "Navigation to an object page",
+    "body": [
+      "this[NavigationMixin.Navigate]({",
+      "    type: 'standard__objectPage',",
+      "    attributes: {",
+      "        actionName: \"${1:home, list, or new}\",",
+      "        objectApiName: \"${2:The API name of the standard or custom object. }\"",
+      "    }",
+      "});"
+    ]
+  },
+  "lwc-nav-record-page": {
+    "prefix": "lwc-nav-record-page",
+    "description": "Navigation to a record page",
+    "body": [
+      "this[NavigationMixin.Navigate]({",
+      "    type: 'standard__recordPage',",
+      "    attributes: {",
+      "        actionName: \"${1:clone, edit, or view}\",",
+      "        recordId: \"${2:recordId}\",",
+      "        objectApiName: \"${3:The API name of the record’s object. Optional for lookups.}\"",
+      "    }",
+      "});"
+    ]
+  },
+  "lwc-nav-record-relationship-page": {
+    "prefix": "lwc-nav-record-relationship-page",
+    "description": "Navigation to a record relationship page",
+    "body": [
+      "this[NavigationMixin.Navigate]({",
+      "    type: 'standard__recordRelationshipPage',",
+      "    attributes: {",
+      "        actionName: \"view\",",
+      "        recordId: \"${1:recordId}\",",
+      "        objectApiName: \"${2:The API name of the record’s object. Optional for lookups.}\",",
+      "        relationshipApiName: \"${3:The API name of the object’s relationship field (optional).}\"",
+      "    }",
+      "});"
+    ]
+  },
+  "lwc-nav-web-page": {
+    "prefix": "lwc-nav-web-page",
+    "description": "Navigation to an external URL",
+    "body": [
+      "this[NavigationMixin.Navigate]({",
+      "    type: 'standard__webPage',",
+      "    attributes: {",
+      "       url: \"${1:The URL of the page you are navigating to.}\"",
+      "    }",
+      "});"
+    ]
+  },
+  "lwc-toast": {
+    "prefix": "lwc-toast",
+    "description": "Creates toast success function",
+    "body": [
+      "this.dispatchEvent(new ShowToastEvent({",
+      "    title: '${1:title}',",
+      "    message: '${2:message}',",
+      "    variant: 'success'",
+      "}));"
+    ]
+  },
+  "lwc-toast-error": {
+    "prefix": "lwc-toast-error",
+    "description": "Creates toast error function",
+    "body": [
+      "this.dispatchEvent(new ShowToastEvent({",
+      "    title: '${1:title}',",
+      "    message: '${2:message}',",
+      "    variant: 'error'",
+      "}));"
+    ]
+  },
+  "lwc-toast-warning": {
+    "prefix": "lwc-toast-warning",
+    "description": "Creates toast warning function",
+    "body": [
+      "this.dispatchEvent(new ShowToastEvent({",
+      "    title: '${1:title}',",
+      "    message: '${2:message}',",
+      "    variant: 'warning'",
+      "}));"
+    ]
+  },
+  "lwc-imperative-apex": {
+    "prefix": "lwc-imperative-apex",
+    "description": "Creates an imperative Apex call",
+    "body": [
+      "${1:method name}()",
```

Comment:
@muenzpraeger Apologies, I'm not familiar with what lwc-imperative-apex is, so I want to double check. Should 'method' here be 'method', or should this be 'function', or have a hyphen between method and name instead?

Reply:
It should be a method name that is exposed through Apex. There are more details in the lwc-recipes repo (soon to be public).